resources/views/campaigns/webhooks/_form.blade.php

```diff
@@ -0,0 +1,53 @@
+{{ csrf_field() }}
+<x-grid type="1/1">
+<x-forms.field
+    field="action"
+    :required="true"
+    :label="__('campaigns.webhooks.fields.event')"
+    >
+    {!! Form::select('action', [1 => __('campaigns.webhooks.fields.events.new'), 2 => __('campaigns.webhooks.fields.events.edited'), 3 => __('campaigns.webhooks.fields.events.deleted')]) !!}
+</x-forms.field>
+
+<x-forms.field
+    field="url"
+    :required="true"
+    :label="__('campaigns.webhooks.fields.url')"
+    >
+    {!! Form::text('url', null, ['placeholder' => __('campaigns.webhooks.placeholders.url'), 'class' => '', 'maxlength' => 200, 'required']) !!}
```

Comment:
max-length is different than the value in the database and will cause errors/data loss